### ISSUE TITLE
Screen reader focus order is not logical on the 'Intune Connector for Active Directory' dialog while navigating using arrow keys in scan mode in windows.

### ISSUE THREAD

**Issue by msftedad** [wix/react-native-navigation#8109]
### What happened?

**User Impact:**
Blind or low vision users may face significant accessibility barriers if the screen reader focus order is not logical on the 'Intune Connector for Active Directory' dialog in scan mode. This can disrupt comprehension and workflow efficiency.

**Pre- Requisite:**
Turn on narrator.

**Actual Result:**
Screen reader focus order is not logical on the 'Intune Connector for Active Directory' dialog while navigating using arrow keys in scan mode in windows.
Current focus: Text field > 'I agree to the...' > 'Option' > 'Install' > Close > Image > 'Intune connector for ...' > 'Intune connector for active directory setup' title > Minimize > Maximize > Close 

**Refer Attachment:**
Screen reader focus order is not logical on the 'Intune Connector for Active Directory' dialog in scan mode in windows.png
Screen reader focus order is not logical on the 'Intune Connector for Active Directory' dialog in scan mode in windows.mp4

<img width="972" height="622" alt="Image" src="https://github.com/user-attachments/assets/110371ac-5530-4909-a973-4f313504299c" />
https://github.com/user-attachments/assets/931d9562-a24e-4513-ad55-eb764d214429

### What was the expected behaviour?

Screen reader focus order should be logical on the 'Intune Connector for Active Directory' dialog while navigating using arrow keys in scan mode in windows.
Current focus:  'Intune connector for active directory setup' title > Minimize > Maximize > Close > 'Intune connector for ...' > Text field > 'I agree to the...' > 'Option' > 'Install' > Close

### Was it tested on latest react-native-navigation?

- [x] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### Help us reproduce this issue!

1. Connect to the virtual machine.
2. Open 'ODJ Connector Bootstrapper' application.
3. 'Intune Connector for Active Directory dialog will be opened.
4. Navigate the 'Intune Connector for Active Directory' dialog in scan mode using arrow keys.
5. Observe whether screen reader focus order is in sequential order on the dialog or not.

### In what environment did this happen?

OS: Windows 11 24H2 (OS Build 26100.4946)
App: ODJ Connector Bootstrapper
Screen Reader: Narrator

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by gosha212** [comment]
I think it is some kind of mistake. This is not related to our project
